modules/ROOT/pages/get-mapping-recommendations.adoc

```diff
@@ -1,9 +1,9 @@
-= Get a Recommendation for Mapping Payloads for a Transformation
+= Get a Recommendation for Mapping Data Types for a Transformation
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In the Transform card, you can view recommendations for how to map the data elements in an input payload to the data elements in an output payload. If a recommendation suits your needs, you can click *Apply* to save the mapping and use it in the card.
+In the Transform card, you can view recommendations for how to map the data elements in an input data type to the data elements in an output type. If a recommendation suits your needs, you can click *Apply* to save the mapping and use it in the card.
```

Comment:
Are input and output data types? Do they need tick marks? `input` and `output` I don't know enough to be sure. 
I would just skip "you can" and say, ...click Apply...